source/Octopus.Tentacle.Client/Scripts/ScriptExecutionOrchestrator.cs

```diff
@@ -141,24 +159,37 @@ async Task<ScriptServiceVersion> DetermineScriptServiceVersionToUse(ILog logger,
         {
             logger.Verbose("Determining ScriptService version to use");
 
-            var tentacleCapabilities = await rpcCallExecutor.ExecuteWithRetries(
-                RpcCall.Create<IClientCapabilitiesServiceV2>(nameof(IClientCapabilitiesServiceV2.GetCapabilities)),
-                ct =>
-                {
-                    return capabilitiesServiceV2.GetCapabilities(new HalibutProxyRequestOptions(ct));
-                },
-                logger,
-                // We can abandon a call to Get Capabilities and walk away as this is not running anything that needs to be cancelled on Tentacle
-                abandonActionOnCancellation: true,
-                clientOperationMetricsBuilder,
-                cancellationToken).ConfigureAwait(false);
+            CapabilitiesResponseV2 tentacleCapabilities;
+
+            if (rpcRetrySettings.RetriesEnabled)
+            {
+                tentacleCapabilities = await rpcCallExecutor.ExecuteWithRetries(
+                    RpcCall.Create<IClientCapabilitiesServiceV2>(nameof(IClientCapabilitiesServiceV2.GetCapabilities)),
+                    ct => capabilitiesServiceV2.GetCapabilities(new HalibutProxyRequestOptions(ct)),
```

Comment:
For other calls the action performed by ExecuteWithRetries and Execute is an action that is reused by both. Should we keep that consistent and do that here as well

Reply:
@acodrington thoughts ⬆️

Reply:
Yeah, good pickup, it should be the same 👍🏻